R/tbl_svysummary.R

```diff
@@ -613,17 +613,11 @@ c_form <- function(left = NULL, right = 1) {
       select(-any_of(c(
         all.vars(x$call$id),
         all.vars(x$call$probs),
-        all.vars(x$call$strata),
```

Comment:
cf. #1171 discussion. There are cases where we want to display `x$call$id`.
And in some (less frequent cases) we need to display statistics about `weights`. 

cf. previous comment. `everything()` should display all variables.

If really there is a need for identifying survey variables, let's create a `survey_vars()` selector and let the user indicates `include = -survey_vars()`